channel/src/main/java/com/synopsys/integration/alert/channel/jira2/cloud/JiraIssueAlertPropertiesManager.java

```diff
@@ -15,11 +15,11 @@
 import com.synopsys.integration.jira.common.model.response.IssuePropertyResponseModel;
 import com.synopsys.integration.jira.common.rest.service.IssuePropertyService;
```

Comment:
Ok, so this class can be used with both Jira Cloud and Jira Server?

Reply:
That was my impression from reading the old code.